src/utils/csrf.js

```diff
@@ -7,7 +7,7 @@ import * as AppConstants from '../app-constants.js'
 
 // defaults to ignore GET, HEAD, OPTIONS
 const { doubleCsrfProtection, generateToken } = doubleCsrf({
-  getSecret: (req) => req.secret || AppConstants.CSRF_SECRET,
+  getSecret: () => AppConstants.CSRF_SECRET,
```

Comment:
We discussed this in eng meeting today, need to make sure we're using this correctly.